jujugui/static/gui/src/app/components/inspector/config/test-config.js

```diff
@@ -647,6 +647,58 @@ describe('Configuration', function() {
     assert.equal(updateUnit.callCount, 0);
   });
 
+  it('stops setting changes if service name is blank', () => {
+    var option1 = { key: 'option1key', type: 'string' };
```

Comment:
can you update these to use `let` and `const` plz.

Reply:
Crap, sorry. Copypasta.